templates/aws_cspm_cloudformation_v2.json

```diff
@@ -209,19 +233,44 @@
     "CrowdStrikeCloudTrail": {
       "Type": "AWS::CloudTrail::Trail",
       "Condition": "ShouldCreateIOACloudTrailResource",
+      "Metadata": {
+        "cfn_nag": {
```

Comment:
Incorrect suppression. Example below. For additional information, refer to [the doc](https://www.checkov.io/2.Basics/Suppressing%20and%20Skipping%20Policies.html).

```
  MyDB:
    Metadata:
      checkov:
        skip:
          - id: "CKV_AWS_157"
            comment: "Ensure that RDS instances have Multi-AZ enabled"
    Type: "AWS::RDS::DBInstance"
    Properties:
```